MonoGame.Framework.Content.Pipeline/Processors/FontDescriptionProcessor.cs

```diff
@@ -197,19 +203,110 @@ static Glyph[] ImportFont(FontDescription options, out float lineSpacing, out in
 
 			return glyphs.ToArray();
 		}
-
+        /// <summary>
+        /// Retrieves the path to the font with the requested filename looking for variations for the style
+        /// </summary>
+        /// <param name="fontname">Filename of the font, without extension</param>
+        /// <param name="fontDirectories">List of possible folders that may contain the font</param>
+        /// <param name="style">Style of the font</param>
+        /// <returns>Full path to the found font file or null.
+        /// When selecting the file to return, the function returns the better matches the requested style.
+        /// If the requested style is bold and italic, but there's no file that has both styles,
+        /// the first option is returning the italic one, then the bold one, and finally the regular one</returns>
+        string FindFontByFileName(string fontname, List<string> fontDirectories, FontDescriptionStyle style)

```

Comment:
I did it to give the project collaborators a quick overview of what the method should do